integration/pages/BasePage.js

```diff
@@ -0,0 +1,44 @@
+const config = require('../utils/config.js');
+const { until } = require('selenium-webdriver');
+
+class BasePage {
+  constructor(driver) {
+    this.driver = driver;
+  }
+
+  async visit(route) {
+    await this.driver.get(config.baseUrl + route);
+  }
+
+  find(locator) {
+    return this.driver.findElement(locator);
+  }
+
+  async findAll(locator) {
+    return await this.driver.findElements(locator);
+  }
+
+  async isDisplayed(locator, timeout) {
```

Comment:
Seem like we always pass timeout when using this function? Do we have a use case for the else part?
Also can we just set 5000 as the default value for timeout here instead of needing to always passing it in?

Reply:
Refactored `isDisplayed` function a little bit. 
And set the default `timeout` to 5000 for all wait until functions.

```js
  // Check if the element is able to be found
  async isFound(locator, timeout = 5000) {
    return await this.driver.wait(until.elementLocated(locator), timeout);
  }

  // Check if the element is `displayed`
  async isDisplayed(locator, timeout = 5000) {
    await this.driver.wait(until.elementIsVisible(this.find(locator)), timeout);
  }
```

So to check if an element appears and displayed, for example:
```js
  async checkIfTestAppHasLoaded() {
    const isFound = await this.isFound(ELEMENTS.appHeader);
    assert(isFound, 'The header of Test App was not found, Test App was not loaded successfully');

    const isDisplayed = await this.isDisplayed(ELEMENTS.appHeader);
    assert(isDisplayed, 'The header of Test App was not displayed, Test App was not loaded successfully');
  }
```